test/test-spawn.c

```diff
@@ -1540,3 +1540,168 @@ TEST_IMPL(spawn_reads_child_path) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+#ifndef _WIN32
+static int mpipe(int *fds)
+{
+  if ( pipe(fds) == -1 ){
```

Comment:
Remove the space between the parenthesis and "pipe". There are more occurrences of this.